docker/Dockerfile

```diff
@@ -155,10 +135,13 @@ RUN mkdir -p $ONEAPI_INSTALL_DIR $ONEAPI_CACHE_DIR $ONEAPI_DOWNLOAD_DIR $ONEAPI_
     && wget -P $ONEAPI_DOWNLOAD_DIR $ONEAPI_INSTALLER_URL/$ONEAPI_INSTALL_BINARY_NAME \
     && chmod +x $ONEAPI_DOWNLOAD_DIR/$ONEAPI_INSTALL_BINARY_NAME \
     && $ONEAPI_DOWNLOAD_DIR/$ONEAPI_INSTALL_BINARY_NAME -a -s --eula accept \
-        --action modify --components $ONEAPI_COMPONENTS  \
+        --action install --components $ONEAPI_COMPONENTS  \
         --install-dir $ONEAPI_INSTALL_DIR --log-dir $ONEAPI_LOG_DIR \
         --download-cache $ONEAPI_CACHE_DIR \
-    && rm -Rf $TMPDIR/*
+    && rm -Rf $TMPDIR/* \
+    && mkdir -p $LLVM_SPIRV_INSTALL_DIR \
+    && mv $ONEAPI_INSTALL_DIR/compiler/latest/linux/bin-llvm/llvm-spirv $LLVM_SPIRV_INSTALL_DIR \
+    && ln -s $LLVM_SPIRV_INSTALL_DIR/llvm-spirv $ONEAPI_INSTALL_DIR/compiler/latest/linux/bin-llvm
```

Comment:
I think the rational of separating `LLVM_SPIRV_INSTALL_DIR` and `ONEAPI_INSTALL_DIR` must be documented.

Reply:
I changed my mind and removed the changes in this block and only extract the `llvm_spirv` binary at the very end (see https://github.com/soda-inria/sklearn-numba-dpex/pull/83).

Adding a comment to explain why.